test/cpp/naming/resolver_component_test.cc

```diff
@@ -87,32 +97,50 @@ bool convert_string_to_bool(std::string bool_str) {
   }
 }
 
-vector<GrpcLBAddress> parse_expected_addrs(std::string expected_addrs) {
+void ExpectTokenAndMoveForward(const char *token, std::string &expected_addrs) {
```

Comment:
References should only be used for `const` parameters.

https://google.github.io/styleguide/cppguide.html#Reference_Arguments